### ISSUE TITLE
Adding the code in appdelegate.h causes a build error [REACT 0.71.10]

### ISSUE THREAD

**Issue by Leli1024** [FormidableLabs/react-native-app-auth#862]
When adding the code provided for appdelegate.h the following build error occurs:

```
fatal error: 'RNAppAuthAuthorizationFlowManager.h' file not found
#import "RNAppAuthAuthorizationFlowManager.h"
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.


** BUILD FAILED **


The following build commands failed:
        CompileC /Users/X/Library/Developer/Xcode/DerivedData/X-eemvneiqlpgwrcfjqvrtgzixgxrc/Build/Intermediates.noindex/FAIE.build/Debug-iphonesimulator/X/Objects-normal/arm64/AppDelegate.o /Users/X/Downloads/X/X/X/X/AppDelegate.mm normal arm64 objective-c++ com.apple.compilers.llvm.clang.1_0.compiler (in target 'X' from project 'X')
(1 failure)
```

**Comment by digitalthjarta** [comment]
It seems like the documented needed changes for iOS, does not reflect newer version of React Native, like RCTAppDelegate and that AppDelegate is an .mm file. 

Can I get help with the correct modern way to set this up on iOS?

**Comment by airimia1995** [comment]
+!

**Comment by donpanpete** [comment]
I follow this https://github.com/FormidableLabs/react-native-app-auth/issues/867

**Comment by usman-inovio** [comment]
@digitalthjarta here's my AppDelegate.mm and AppDelegate.h hope you find help from this 

--------------------------
#import "AppDelegate.h"

#import <React/RCTBundleURLProvider.h>
#import <React/RCTLinkingManager.h>
@implementation AppDelegate

- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
{
  self.moduleName = @"app";
  // You can add your custom initial props in the dictionary below.
  // They will be passed down to the ViewController used by React Native.
  // They will be passed down to the ViewController used by React Native.
  self.initialProps = @{};
  
   if ([self.userActivity.activityType isEqualToString:NSUserActivityTypeBrowsingWeb]) {
     if (self.authorizationFlowManagerDelegate) {
       BOOL resumableAuth = [self.authorizationFlowManagerDelegate resumeExternalUserAgentFlowWithURL:self.userActivity.webpageURL];
       if (resumableAuth) {
         return YES;
       }
     }
   }

  return [super application:application didFinishLaunchingWithOptions:launchOptions];
}

- (NSURL *)sourceURLForBridge:(RCTBridge *)bridge
{
#if DEBUG
  return [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:@"index"];
#else
  return [[NSBundle mainBundle] URLForResource:@"main" withExtension:@"jsbundle"];
#endif
}

/// This method controls whether the `concurrentRoot`feature of React18 is turned on or off.
///
/// @see: https://reactjs.org/blog/2022/03/29/react-v18.html
/// @note: This requires to be rendering on Fabric (i.e. on the New Architecture).
/// @return: `true` if the `concurrentRoot` feature is enabled. Otherwise, it returns `false`.
- (BOOL)concurrentRootEnabled
{
  return true;
}

@end


-----------------









#import <RCTAppDelegate.h>
#import <UIKit/UIKit.h>
#import "RNAppAuthAuthorizationFlowManager.h"

@interface AppDelegate : RCTAppDelegate <RNAppAuthAuthorizationFlowManager>
// @interface AppDelegate : UIResponder < RCTAppDelegate, RNAppAuthAuthorizationFlowManager>

@property(nonatomic, weak)id<RNAppAuthAuthorizationFlowManagerDelegate>authorizationFlowManagerDelegate;

@end

**Comment by robwalkerco** [comment]
Closed by #900
